src/components/vm/nics/vmNicsCard.tsx

```diff
@@ -240,15 +242,26 @@ const NetworkSource = ({
                     {network.target}
                 </DescriptionListDescription>
             </DescriptionListGroup>}
-        </DescriptionList>
+        </>
     );
 };
 
-interface VmNetworkTabProps {
-    vm: VM,
-    networks: Network[],
-    onAddErrorNotification: (notification: Notification) => void;
-}
+const NetworkSourceAbbrev = ({
+    network,
+    id,
+} : {
+    network: VMInterface,
+    id: string,
+}) => {
+    const source_name = getIfaceSourceName(network);
+
+    if (!source_name)
+        return null;
+    else if (typeof source_name == "string")
+        return <div id={id}>{source_name}</div>;
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2324-02ce13bf-20250922-141831-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)